pkg/doc/did/service.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package did
+
+// Service represents any type of service the entity wishes to advertise
+type Service map[string]interface{}
```

Comment:
Is there a case for a non-string storage in Service?

Are there mandatary fields that would be best represented as a struct?

Reply:
done